{{cookiecutter.repo_slug}}/.github/labels.yml

```diff
@@ -1,7 +1,7 @@
 ---
 - name: "breaking-change"
   color: ee0701
-  description: "A breaking change for existing users."
+  description: "A change that changes the API or breaks backward compatibility for users."
```

Comment:
So! "Breaking change" is actually not a well-defined term. Historically (like pre-Internet software) it was mostly about backwards compatibility. In the mid-to-late 2000s it came to encompass changes to APIs, which would break anything that was consuming it. Conversationally it's often used for any user interface change (e.g. if your project had a command `foo` and removed it, that would be a breaking change). I'm not sure you can address all of those possible usages in the description; really it's up to the project maintainer to explain what they mean. But I think just restating the term in a longer form doesn't really help, so this is my attempt to at least add some information in the description.